lib/model.js

```diff
@@ -2426,6 +2426,7 @@ class Model {
    * @param  {Function}     [options.logging=false]                       A function that gets executed while running the query to log the sql.
    * @param  {boolean}      [options.benchmark=false]                     Pass query execution time in milliseconds as second argument to logging function (options.logging).
    * @param  {string}       [options.searchPath=DEFAULT]                  An optional parameter to specify the schema search_path (Postgres only)
+   * @param  {object}       [options.conflictWhere]                       An optional parameter that specifies a where clause for the `ON CONFLICT` part of the query (in the same format as the where option for a finder such as `findByPk`). Postgres/SQLite only. (note: updateOnDuplicate must be present for this to be used.)
```

Comment:
We should consider logging a warning if using `conflictWhere` without `updateOnDuplicate`, since this might be overseen by some users, even though this note is added.

Reply:
:+1: worth considering.  It's worth noting that in the future we might use other methods after `ON CONFLICT` (such as `DO NOTHING`) which would still use this.

Reply:
oh wait this is specifically for Model.upsert - nevermind. `updateOnDuplicate` is for `bulkCreate` upserts.